app/src/main/java/edu/grinnell/grinnell_publications_android/Views/MainActivity.java

```diff
@@ -42,15 +53,52 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 //Toggle to fragment associated with clicked menu item
                 switch (menuItem.getItemId()){
                     case R.id.all_publications:
+                        AllPublications allPublications = new AllPublications();
+                        replaceFrameWithFragment(allPublications);
+                        return true;
+
+                    case R.id.bookmarks:
+                        Bookmarks bookmarks = new Bookmarks();
+                        replaceFrameWithFragment(bookmarks);
+                        return true;
 
+                    case R.id.my_publications:
+                        MyPublications publications = new MyPublications();
+                        replaceFrameWithFragment(publications);
+                        return true;
+
+                    default:
+                        Toast.makeText(getApplicationContext(), R.string.transaction_error, Toast.LENGTH_LONG).show();
```

Comment:
We'll probably want to use a snackbar instead of toasts when displaying error. A wrapper around snackbar (for our needs) would be good.